### PR TITLE
ft-230 Added refactor request template

### LINKED ISSUE #230: [Feature]: Create Refactoring Issue Template
For developers

### PR DESCRIPTION
**Pull request by Njima1572**
## Which Issue was this PR made for?

- Closes #230 

## What is within the scope with this PR

Creating a template for refactor request

## Other sidenotes that reviewer should be aware of

Got it from here and modified it a little bit

https://github.com/inovex/elements/blob/master/.github/ISSUE_TEMPLATE/refactor-request.md

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/refactor_request.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    attributes:
+      value: |
+        Thanks for the feature request!
+  - type: textarea
+    id: request
+    attributes:
+      label: Refactor Request
+      description: Is your refactor request related to a problem? Please describe.
+      placeholder: A clear and concise description of what the problem is. Ex. I'm having problems reading the code because [...] (nice to have file/line references too)
+      value: ""
```
Comment (siberowl) [inline .github/ISSUE_TEMPLATE/refactor_request.yaml:16]: I agree not setting a default value, it was a bit annoying removing it every time.

Comment (siberowl) [inline .github/ISSUE_TEMPLATE/refactor_request.yaml:16]: I agree not setting a default value, it was a bit annoying removing it every time (although looks like it's getting a warning of some sort)
image

**Review by siberowl — APPROVED** [review]
I think this is ft-230 instead of 231 but other than that looks great.

File: .github/ISSUE_TEMPLATE/refactor_request.yaml (hunk 1 above)
Comment (Njima1572) [reply .github/ISSUE_TEMPLATE/refactor_request.yaml:16]: how did you get this warning??

Comment (siberowl) [reply .github/ISSUE_TEMPLATE/refactor_request.yaml:16]: I think I looked at the file from the branch (not the pr)

**Review by siberowl — APPROVED** [review]
LGTM!

**Pull request merged**